src/main/java/com/example/shorturl/service/url/ShortenUrlService.java

```diff
@@ -39,7 +39,7 @@ public UrlEntity shortUrl(UrlRequest urlRequest) {
                 .id(sequenceGeneratorService.getSequenceNumber(SEQUENCE_NAME))
                 .shortUrl(encodedUrl)
                 //TODO: Make dynamic
-                .completeShortUrl("http://localhost:8080/api/v1/url/redirect/" + encodedUrl)
+                .completeShortUrl("http://localhost:8080/api/redirect/" + encodedUrl)
```

Comment:
Should it be hardcoded? Can we retrieve value from the context somewhere? Because, if we have another port or will deploy it anywhere, it will still stay as a localhost which is not acceptable